roles/acm_hypershift/tasks/download-cli.yml

```diff
@@ -1,9 +1,13 @@
 ---
 - name: Download stable openshift client
   vars:
-    ocp_client_url: "https://mirror.openshift.com/pub/openshift-v4/x86_64/clients/ocp/stable-4.14/openshift-client-linux.tar.gz"
+    ocp_client_url: >
+      https://mirror.openshift.com/pub/openshift-v4/
+      x86_64/clients/ocp
```

Comment:
I like the split, and I think you could include the lines 9 and 10 here as well so you build the client url in a single place.

Reply:
Thanks